app/controllers/bets_controller.rb

```diff
@@ -1,6 +1,9 @@
 class BetsController < ApplicationController
   def index
-    @bets = Bet.all
+    @items_per_page = 25
+    @current_page = params[:page] || '1'
+    @bets = Bet.offset(@items_per_page * (@current_page.to_i - 1))
```

Comment:
This is :+1: for now,
But we might want to move this into some kind of helper if we ever want to paginate any other resources. That way we can share this code between controllers